### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by MatthewKaplan**
- [ ] Check this if the PR has been approved by the team to be a quick patch and the rest of this form will not be filled out
​
## What functionality does this accomplish?
closes #
​
**Description:**

`Initial setup for using navigation with react native. I created two new components SignUpForm and LoginForm and used navigation to get to both components from the splash page.`
​
​
## What did you struggle on to complete?
​
​
​
### Test Coverage Percentage: 0%
- [x] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened):
​
## Checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code
- [ ] I have partially tested my code (please explain why):
​
## Helpful Resources:
- Resource link AND small description of info gathered/learned
​
​
​
## Review Requests(optional):
​
​
​
## Please include an emoji of how you feel about this branch: 🎸

### CODE REVIEW

File: src/Router.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Scene, Router } from 'react-native-router-flux';
```
Comment (MaxBSilver) [inline src/Router.js]: Is this a similar Router to what he have been using or is this is Navigator?

File: src/components/LoginForm/LoginForm.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
+
+class LoginForm extends Component {
+  render() {
+    return (
+      <View>
+        <Text> LoginForm </Text>
```
Comment (MaxBSilver) [inline src/components/LoginForm/LoginForm.js]: Is there a reason for the multiple <Text> duplicates?

File: src/Router.js (hunk 1 above)
Comment (MatthewKaplan) [reply src/Router.js]: It's different but not too difficult at all, I run you through it during work time today.

File: src/components/LoginForm/LoginForm.js (hunk 2 above)
Comment (MatthewKaplan) [reply src/components/LoginForm/LoginForm.js]: Nope just so I can see a lot of text on the screen when I navigated to another component.

**Pull request merged**